src/Components/Patient.js

```diff
@@ -0,0 +1,27 @@
+import React, { useState } from 'react';
+
+export default function Patient(props) {
+  const [ click, setClick ] = useState();
+  
+  const handleClick = () => {
+    const option = click ? setClick(false) : setClick(true)
+    return option
+  }
+
+  const display = click ? true : 'none'
+  
+  return (
+    <div className="Patient" style={{border: '3px solid black', margin: '5px'}}>
```

Comment:
Although it's quick and easy, I wouldn't add styling to your components like this!